CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ at anytime.
   * Added a new startup stage to indicate if the daemon is waiting for the `wallet_unlock` command.
   * Add `--conf` CLI flag to specify an alternate config file
   * Added `blockchain_name` and `lbryum_servers` to the adjustable settings
+  * Added abandon related information to `transaction_list` command
```

Comment:
What abandon information was added? The changelog should tell users of the api what they need to know to utilize the new behavior